src/components/styles/book.css

```diff
@@ -1,19 +1,16 @@
 .book-card {
-  border: 1px solid #e8e8e8;
-  margin-top: 40px;
-  padding: 2rem;
-  background-color: #fff;
-  border-radius: 4px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
 }
 
 .book-info {
   display: flex;
   flex-direction: column;
```

Comment:
- [ ] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code to No repeating of the same rule by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```css
.d-flex {
  display: flex;
}

.flex-center {
  flex-direction: column;
  align-items: center;
}
```
- JS file
```js
<div className="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

Reply:
@done